.github/workflows/build.yml

```
@@ -331,3 +331,54 @@ jobs:
           name: msys2-${{matrix.boards}}-builds
           path: buildartifacts/
         continue-on-error: true
+
+  # Select the msvc Builds based on PR Arch Label
+  msvc-Arch:
+    uses: apache/nuttx/.github/workflows/arch.yml@master
+    needs: Fetch-Source
+    with:
+      os: Linux
```

Let's change "Linux" to "msvc", so (in future) we can run special Build Rules in `arch.yml`
```suggestion
      os: msvc
```

---

thanks,Done